code/game/turfs/closed/walls.dm

```diff
@@ -86,79 +96,21 @@
 	if(girder_type)
 		new /obj/item/stack/sheet/iron(src)
 
-/turf/closed/wall/ex_act(severity, target)
```

Comment:
All cases of stuff like this were moved to the integrity system instead so are deleted